crates/environ/src/vmoffsets.rs

```diff
@@ -15,13 +15,13 @@
 //      gc_heap_data: *mut T, // Collector-specific pointer
 //      store: *mut dyn Store,
 //      type_ids: *const VMSharedTypeIndex,
+//      imported_memories: [VMMemoryImport; module.num_imported_memories],
```

Comment:
Could you add a comment here saying somthing like "Put memories close to the front, so instructions referencing them can use smaller immediate fields, as load/store instructions tend to be frequent"?